### ISSUE TITLE
Intro to Neural Networks - usage of "error" term

### ISSUE THREAD

**Issue by AlphaGit** [udacity/dlnd-issue-reports#56]
Across the texts and explanations I've found a very loose usage of the term "error". Sometimes it might refer to the absolute difference between the target and output values. Sometimes it is used to be referred to refer to the error gradient (which also considers the weights and derivative function). 

The concept is still a little bit fuzzy to me, as I'm trying to grasp it, but I believe having the explanations refer to a specific and hardened concept would help better.

Thanks!

Also, see [this forum post](https://discussions.udacity.com/t/different-uses-of-error-in-explanations-is-confusing/215874), also related to the suggestion.

**Comment by jillsthebest** [comment]
I completely agree with this. I just opened an issue called Content improvement that talks about a similar issue:  "error" is not the only term that is used too casually in this course.

**Comment by dhilgarth** [comment]
Agreed as well. Example: Intro To Neural Network 14, Backpropagation. I posted it in the feedback slack channel: https://nd101.slack.com/files/daniel.h/F414HAHC5/pasted_image_at_2017_02_04_04_40_pm.png

**Comment by onemodtwo** [comment]
Wish I had noticed this before posting the same criticism (about the same word) re Intro to NN. - 11. Glad to see I'm in good company.

**Comment by clapollo** [comment]
We had a few issues open related to this. We've made several changes over time to this content, hopefully for the better. I'm closing this issue, but if you still see specific cases of terms used in confusing or conflicting ways, please create a new issue with the details. Thank you.
